Library/Homebrew/cask/test/test_helper.rb

```diff
@@ -28,15 +28,18 @@ def shutup
   if ENV.key?("VERBOSE_TESTS")
     yield
   else
+    err = $stderr.dup
+    out = $stdout.dup
+
     begin
-      tmperr = $stderr.clone
-      tmpout = $stdout.clone
-      $stderr.reopen "/dev/null", "w"
-      $stdout.reopen "/dev/null", "w"
+      $stderr.reopen("/dev/null")
+      $stdout.reopen("/dev/null")
       yield
     ensure
-      $stderr.reopen tmperr
-      $stdout.reopen tmpout
+      $stderr.reopen(err)
+      $stdout.reopen(out)
+      err.close
+      out.close
     end
   end
 end
```

Comment:
I somehow missed that https://github.com/Homebrew/brew/blob/bd1ded93ec46b2ae9c37dbf17dd6d74d3f727440/Library/Homebrew/cask/spec/support/shutup_helper.rb exists.